docs/devops.md

```diff
@@ -0,0 +1,237 @@
+---
+name: ' DevOps practices'
+menu: Best practices
+route: /devops
+---
+
+# DevOps practices
+
+---
+
+## Dockerfile guidelines
+
+From Docker's [_Best practices for writing Dockerfiles_](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/):
+
+> Docker builds images automatically by reading the instructions from a Dockerfile -- a text file that contains all commands, in order, needed to build a given image. A Dockerfile adheres to a specific format and set of instructions which you can find at Dockerfile reference.
+> 
+> A Docker image consists of read-only layers each of which represents a Dockerfile instruction. The layers are stacked and each one is a delta of the changes from the previous layer.
+
+### TL;DR
+
+*   Use the provided [base images](https://hub.docker.com/u/helsinkitest).
+
+*   Keep the image slim.
```

Comment:
Definition of slim?
I've been bitten by images missing basic tools like "ping", or file pager. Sure, you should not need those. In practice I've been forced to add those, to debug weird things happening.